ci/sql/fix-default-namespace.py

```diff
@@ -0,0 +1,28 @@
+import os
+import asyncio
+from gear import Database
+
+
+async def main():
+    default_namespace = os.environ['HAIL_NAMESPACE']
+
```

Comment:
I know this check is probably right, but is it worth explicitly making sure that we're not in the SCOPE = deploy as well just in case the default_namespace is always default. If you've tested that the namespace here is not always "default" (I remember there was an issue with this and that's why we have the variable `is_test_deployment`), then feel free to ignore.

Reply:
I don't think the scope is what we want here -- a test CI can do a `deploy` into its own namespace, which is when `is_test_deployment` would be useful, but in either case it's the namespace that is relevant here not which scope we're in. This fixes all namespaces that are not named `default`. I'm confident that this variable represents the namespace the database is created for.